docs/code_of_conduct.md

```diff
@@ -1,3 +1,5 @@
+[👈 Back to README](../README.md)
```

Comment:
My only nit here is that we might want to leave this at the top level along with things like CONTRIBUTING.md, because I've noticed a few other projects doing it as an ad-hoc practice. But, it's not something that gets special treatment from GitHub, so the case for organizing this into the docs dir is just as valid IMO.